extractors/python_functions/person_extraction/code_snippet.md

```diff
@@ -1,10 +1,8 @@
 ```python
-from typing import List, Tuple
-
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "name"
 
-def person_extraction(record) -> List[Tuple[str, int, int]]:
+def person_extraction(record):
```

Comment:
extractor

Reply:
See other comments about the extraction naming.